### PR TITLE
Added parameter -OutputPath

### PR DESCRIPTION
**Pull request by emilguden**
Added the parameter -OutputPath.
Build-Bicep -Path .\vnet.bicep -OutputPath .\virtualNetork.json

### CODE REVIEW

**Comment by StefanIvemo** [comment]
Fixes #109

**Comment by StefanIvemo** [comment]
@emilguden Looks good overall. Can you add a check that verifies that the input provided to `-Path` is a single `.bicep` file? Right now it is possible to do this:

`Build-Bicep -Path c:\allMyBicepFiles\ -OutputPath c:\armTemplates\template.json`

And it will loop through all `.bicep` files in the folder and output only one `template.json`, because it will be overwritten for each iteration.  We should only support this for single `.bicep` files.

**Comment by emilguden** [comment]
> @emilguden Looks good overall. Can you add a check that verifies that the input provided to `-Path` is a single `.bicep` file? Right now it is possible to do this:
> 
> `Build-Bicep -Path c:\allMyBicepFiles\ -OutputPath c:\armTemplates\template.json`
> 
> And it will loop through all `.bicep` files in the folder and output only one `template.json`, because it will be overwritten for each iteration. We should only support this for single `.bicep` files.

Added ParameterSetName so only -OutputPath or -OutputDirectory can be used + your request for only supporting one file when -OutputPath is used.

**Comment by StefanIvemo** [comment]
@emilguden can you update the docs and help as well?

File: Docs/Help/Build-Bicep.md [hunk 1]
```diff
@@ -49,7 +52,7 @@ Build-Bicep -Path vnet.bicep
 
 ### Example 2: Compile single bicep file and specify the output directory
 ```powershell
-Build-Bicep -Path 'c:\bicep\modules\vnet.bicep' - -OutputDirectory 'c:\armtemplates\vnet.bicep'
+Build-Bicep -Path 'c:\bicep\modules\vnet.bicep' -OutputDirectory 'c:\armtemplates\vnet.bicep'
```
Comment (StefanIvemo) [inline Docs/Help/Build-Bicep.md:55]: While you´re at it. Can you fix this line. It should be:
 `Build-Bicep -Path 'c:\bicep\modules\vnet.bicep' -OutputDirectory 'c:\armtemplates\'`

File: Source/Public/Build-Bicep.ps1 [hunk 2]
```diff
@@ -27,15 +39,20 @@ function Build-Bicep {
...
             testModuleVersion
         }
         if ($PSBoundParameters.ContainsKey('OutputDirectory') -and (-not (Test-Path $OutputDirectory))) {
             $null = New-Item $OutputDirectory -Force -ItemType Directory -WhatIf:$WhatIfPreference
         }
+        if ($PSBoundParameters.ContainsKey('OutputPath') -and (-not (Test-Path $OutputPath))) {
+            $null = New-Item (Split-Path -Path $OutputPath) -Force -ItemType Directory -WhatIf:$WhatIfPreference
+        }
+        if ($PSBoundParameters.ContainsKey('OutputPath') -and ((Split-path -path $Path -leaf) -notmatch "\.bicep$")) { 
+            Write-Error 'If -Path and -OutputPath is used, only one .bicep file can be used, e.g. -Path "C:\Output\template.bicep" -OutputPath "C:\Output\newtemplate.json"'
```
Comment (StefanIvemo) [inline Source/Public/Build-Bicep.ps1:53]: Can we change this to:
'If -Path and -OutputPath parameter is used, only one .bicep file can be used as input to -Path. E.g. -Path "C:\Output\template.bicep" -OutputPath "C:\Output\newtemplate.json"'

File: Source/Public/Build-Bicep.ps1 [hunk 3]
```diff
@@ -6,18 +6,30 @@ function Build-Bicep {
         [Parameter(ParameterSetName = 'Default', Position = 1)]
         [Parameter(ParameterSetName = 'AsString', Position = 1)]
         [Parameter(ParameterSetName = 'AsHashtable', Position = 1)]
+        [Parameter(ParameterSetName = 'CouldBeDefaultButItsNot', Position = 1)]
```
Comment (StefanIvemo) [inline Source/Public/Build-Bicep.ps1:9]: Can we change the parameter set name to OutputPath instead?

**Review by StefanIvemo — APPROVED** [review]

**Pull request merged**